app/instance.py

```diff
@@ -261,5 +261,14 @@ def internal_server_error(error):
                          {'Content-Type': 'application/vnd.api+json'})
 
 
+@app.errorhandler(429)
+def ratelimit_handler(error):
+    dict_ = {'status': 429,
+             'title': 'Request Limit Exceeded',
+             }
+    return make_response(json.dumps(dict_), 429,
+                         {'Content-Type': 'application/vnd.api+json'})
```

Comment:
Properly format and indent. Remove dict_ variable

Reply:
Okay